test/unit/model/NewSqlResultStorageTest.php

```diff
@@ -51,4 +52,28 @@ public function microTimeToTest()
             [1234567890.12345600, new \DateTime('2009-02-13T23:31:30.123456Z')],
         ];
     }
+
+    /**
+     * @param string $test
+     * @param $object
+     * @param string $item
+     */
+    protected function assertItemVariableBasics(string $test, $object, string $item): void
```

Comment:
Are you going to use these methods anywhere?

Reply:
Please see parent class